crates/sandbox/Cargo.toml

```diff
@@ -16,8 +16,20 @@ keywords = ["runtime", "kubernetes", "cri", "container", "pod"]
 categories = ["network-programming", "api-bindings"]
 
 [dependencies]
+async-trait = "0.1.51"
 thiserror = "1.0.29"
 bitflags = "1.3.2"
 common = { path= "../common" }
 derive_builder = "0.10.2"
+dyn-clone = "1.0.4"
 getset = "0.1.1"
+tokio = { version = "1.11.0", features = ["process"] }
```

Comment:
I agree with the use of `tokio`. BTW did you do any comparative study? I'd simply like to know if there is a good reason.

Reply:
I chose tokio because it was already used in other parts of the project and we are already using other components from tokio like tonic and prost. Being able to use tokio console for debugging seemed enticing as well.

Reply:
:+1: